src/ConfigurationsProvider.tsx

```diff
@@ -49,13 +65,23 @@ export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) =
     skip: !Object.keys(sue).length
   });
 
+  const { data: filterData } = useQueryWithApollo(getQuery(QUERY_TYPES.RESOURCE_FILTERS), {
```

Comment:
like `sueConfig` and `sueConfigData` please do the same for `filterData` with `resourceFiltersData`

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a